docs/protocol/extension_statistics.md

```diff
@@ -111,8 +112,15 @@ $model_stat =
   DOES NOT include cache hits.
 
 - "inference_stats" : The aggregate statistics for the
-  model/version. So, for example, "inference_stats":"success"
-  indicates the number of successful inference requests for the model.
+  model. So, for example, "inference_stats":"success" indicates the number of
+  successful inference requests for the model.
+
+- "response_stats" : The aggregate decoupled response statistics for the
+  model. For example, { "key" : { "response_stats" : "success" } } indicates the
```

Comment:
Can we do a real example here?  `key` is still confusing. Something like:
Assume there were `N` number of requests sent to server. An example of a possible response statistics is ...,
where <actual value> is a `key`, which helps to differentiate between decoupled responses generated by the model. It is advised ...

Reply:
Sure, added a brief example for key.